overlays/custom-packages/default.nix

```diff
@@ -5,6 +5,7 @@
 #
 _: {
   nixpkgs.overlays = [
+    (import ./powercontrol)
```

Comment:
This can be removed with https://github.com/tiiuae/ghaf/pull/406/files#r1450579081

Reply:
Done